workflow/snakefile

```diff
@@ -31,7 +32,6 @@ def get_otoole_data(otoole_config: str, var: str) -> list[str]:
         "StorageLevelYearStart",
         "StorageLevelYearFinish",
         "Trade",
-        "DiscountedOperatingCost",
```

Comment:
This removal caused the workflow on my end to not run (I think). I've added a commit to revert this, if that's incorrect let me know.

Reply:
Good catch!